src/Sendables/Email/PhpMailEmailProvider.php

```diff
@@ -30,7 +30,8 @@ public function send(Sendable $email)
             $email->getRecipientList(),
             $email->getSubject(),
             $email->getBodyRaw(),
-            $email->getMailHeadersAsString()
+            $email->getMailHeadersAsString(),
+            "-f".(string)$this->getSender()
```

Comment:
I don't think you want the string conversion of that, because if the EmailAddress has a name associated, it'll return it as e.g. `Cahir O'Doherty <codoherty@gcdtech.com>`, which won't work in the param. You just want `$this->getSender()->email`.